### PR TITLE
gn: Fix build

### PR DESCRIPTION
**Pull request by ShabbyX**
New headers not listed in gn file.

### CODE REVIEW

File: BUILD.gn [hunk 1]
```diff
@@ -82,6 +82,8 @@ config("vulkan_layer_config") {
 
 core_validation_sources = [
   "layers/android_ndk_types.h",
+  "layers/base_node.h",
+  "layers/buffer_state.h",
```
Comment (ncesario-lunarg) [inline BUILD.gn:86]: Does GN require this list be in alphabetical order?

**Review by ncesario-lunarg — APPROVED** [review]
LGTM!

File: BUILD.gn (hunk 1 above)
Comment (ShabbyX) [reply BUILD.gn:86]: No, it was a little bit of OCD on my part.

Our autoformatter does sort them, though we don't run that on third party repos.

Comment (ncesario-lunarg) [reply BUILD.gn:86]: :+1: just curious :)

**Pull request merged**